server/views/pages/adjudicationForReport/formsTab.njk

```diff
@@ -39,6 +39,13 @@
 
     {% set rows = [] %}
     {% for report in reports %}
+      {% for disIssue in report.formattedDisIssueHistory %}
```

Comment:
In the designs the table is displayed ascending hence me putting the history (if there is any) before the latest issuance. Also checked in the API and [ReportedAdjudicationBaseService returns](https://github.com/ministryofjustice/hmpps-manage-adjudications-api/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/reported/ReportedAdjudicationBaseService.kt#L252-L258) `DisIssueHistory` in ascending order so we can simply display it in the order we get it 👌